config/initializers/dfe_analytics.rb

```diff
@@ -11,7 +11,7 @@
 
   # Whether to run entity table checksum job.
   #
-  config.entity_table_checks_enabled = true
+  config.entity_table_checks_enabled = Settings.google.entity_table_checks_enabled
```

Comment:
In relation to 

https://github.com/DFE-Digital/dfe-analytics/blob/80015465040513020d0c2b3a2ae45d4f05f3b547/lib/dfe/analytics/entity_table_check_job.rb#L10

https://github.com/DFE-Digital/register-trainee-teachers/blob/e9d59922583021d03e2522503a4241cbcd306a14/config/sidekiq_cron_schedule.yml#L52-L56

![image](https://github.com/DFE-Digital/register-trainee-teachers/assets/470137/669b1f2a-5e68-478b-8d4b-5225aab22653)